lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -117,6 +23,19 @@
     border-radius: 4px;
     color: #444;
   }
+
+  .slideshow__slide .container > .wrapper {
+    margin: 12px;
```

Comment:
@lkpttn 24px looks fine on my computer, mind if we just switcheroo to that for now?